### PR TITLE
Remove support email

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ The ORCID Registry from its start has been imagined, defined, and developed by m
 
 * **Bug reports should be made in the iDeas Forum or directly to the ORCID team**
 
-    Report the bug to our support team for investigation; once confirmed, we add it to the Bugs board to be addressed. You can report the bug by posting in the [iDeas Forum](https://support.orcid.org/hc/en-us/community/topics) or by submitting a ticket via [our webform](https://orcid.org/help/contact-us) or via [email](mailto:support@orcid.org).
```
Comment (leomendoza123) [inline CONTRIBUTING.md]: Just removed the email reference as the https://orcid.org/help/contact-us was already present

File: orcid-web/src/main/resources/freemarker/includes/ng2_templates/link-account-ng2-template.ftl [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
             <p class="alert">${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.1")}<span *ngIf="signinData.accountId">${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.2")}{{signinData.accountId}}${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.3")}</span>${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.4")}</p>
-            <p>${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.1")}<a href="mailto:{{signinData.institutionContactEmail}}?cc=support@orcid.org&amp;subject=${springMacroRequestContext.getMessage("social.link.unsupported.email.subject")?url}&amp;body=${springMacroRequestContext.getMessage("social.link.unsupported.email.body")?url}">${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.2")}</a>${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.3")}<a href="{{getBaseUri()}}/signin">${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.4")}</a>${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.5")}</p>
```
Comment (leomendoza123) [inline orcid-web/src/main/resources/freemarker/includes/ng2_templates/link-account-ng2-template.ftl]: @lizkrznarich Just removed the CC to support@orcid.org, do we want to keep it?

**Comment by leomendoza123** [comment]
A list of all the texts changed with their textual context can be found here 

https://docs.google.com/spreadsheets/d/1nT8xB_qPRHGHbZSl1cZy97ljL-1j2xzy-R129RMCjRg/edit#gid=1699973519

**Comment by leomendoza123** [comment]
[Registry: Remove reference to support@ email address (3)](https://trello.com/c/BCW5Op5d/5824-registry-remove-reference-to-support-email-address-3)

File: orcid-web/src/main/resources/freemarker/includes/ng2_templates/link-account-ng2-template.ftl (hunk 2 above)
Comment (lizkrznarich) [reply orcid-web/src/main/resources/freemarker/includes/ng2_templates/link-account-ng2-template.ftl]: @leomendoza123 I think this is the one exception where we should keep support@orcid.org bc we do want users to copy us when they contact their IdP admin about institutional signin.

**Pull request merged**